delete-serverless-service/index.js

```diff
@@ -76,7 +76,6 @@ module.exports.handler = (event, context, cb) => {
 		params.domain = event.body.domain;
 		params.version = version;
 		params.db_service_id = event.body.id;
-        params.auth_token = event.headers.Authorization
```

Comment:
Since we are removing this here, can we 

- remove the same during request validation?
- update unit tests?

Reply:
Removed the request validation for headers and also updated the test case